scripts/build.js

```diff
@@ -1,14 +1,19 @@
 const ncc = require("../src/index.js");
-const { statSync, writeFileSync, readFileSync, unlinkSync } = require("fs");
+const { statSync, writeFileSync, readFileSync, unlinkSync, rmdirSync } = require("fs");
 const { promisify } = require("util");
-const { relative } = require("path");
+const { relative, join } = require("path");
 const copy = promisify(require("copy"));
 const glob = promisify(require("glob"));
 const bytes = require("bytes");
 
 const minify = true;
+const v8cache = true;
+const cache = join(__dirname, "..", ".cache");
```

Comment:
We shouldn't need to locate a custom cache directory - we can just use the same one used in the default build as that is already versioned.

Reply:
I like keeping this to the local project since the default ncc cache is a tmp dir like `/var/folders/v7/7kzxdgb81x324tk9pg633s200000gm/T/ncc-cache` so it makes it hard to inspect. But putting it in the project root makes it easier to inspect.

<img src="https://user-images.githubusercontent.com/229881/116472782-5b6c5180-a844-11eb-94b3-094e0a32f057.png" width=200 />

Reply:
I see ok.